frontend/public/components/resource-quota.jsx

```diff
@@ -13,6 +13,42 @@ const menuActions = [...common];
 
 const quotaKind = quota => quota.metadata.namespace ? referenceForModel(ResourceQuotaModel) : referenceForModel(ClusterResourceQuotaModel);
 
+const quotaResourceLabels = Object.freeze({
+  'configmaps': {label: 'config maps', titleCase: 'Config Maps'},
+  'cpu': {label: 'cpu request', titleCase: 'CPU (Request)'},
+  'limits.cpu': {label: 'cpu limit', titleCase: 'CPU (Limit)'},
+  'limits.memory': {label: 'memory limit', titleCase: 'Memory (Limit)'},
+  'memory': {label: 'memory request', titleCase: 'Memory (Request)'},
+  'openshift.io/imagesize': {label: 'image size', titleCase: 'Image Size'},
+  'openshift.io/imagestreamsize': {label: 'image stream size', titleCase: 'Image Stream Size'},
+  'openshift.io/projectimagessize': {label: 'project image size', titleCase: 'Project Image Size'},
+  'persistentvolumeclaims': {label: 'persistent volume claims', titleCase: 'Persistent Volume Claims'},
+  'requests.storage': {label: 'storage request', titleCase: 'Storage (Request)'},
+  'pods': {label: 'pods', titleCase: 'Pods'},
+  'replicationcontrollers': {label: 'replication controllers', titleCase: 'Replication Controllers'},
+  'requests.cpu': {label: 'cpu request', titleCase: 'CPU (Request)'},
+  'requests.memory': {label: 'memory request', titleCase: 'Memory (Request)'},
+  'resourcequotas': {label: 'resource quotas', titleCase: 'Resource Quotas'},
+  'secrets': {label: 'secrets', titleCase: 'Secrets'},
+  'services': {label: 'services', titleCase: 'Services'},
+  'services.loadbalancers': {label: 'service load balancers', titleCase: 'Service Load Balancers'},
+  'services.nodeports': {label: 'service node ports', titleCase: 'Service Node Ports'},
```

Comment:
@dtaylor113 This looks like an incomplete list.

https://docs.okd.io/latest/admin_guide/quota.html#managed-by-quota

I'm worried that this is constantly going to get out of date, so we have a mix of humanized / unhumanized labels. We might skip this and just show the real value. It's also confusing to see something different in the table than you need to add the to YAML.

cc @cshinn

Reply:
@spadgett, @cshinn, so instead of:
![image](https://user-images.githubusercontent.com/12733153/49018834-6e056c80-f15a-11e8-9760-e0fd0ee8f063.png)
We'd show:
```
    limits.cpu
    requests.cpu
    limits.memory
    requests.memory
    pods
```

Reply:
@dtaylor113 I think that's better because we're never going to be able to humanize everything, and things will always be added later. Storage classes follow the format `<storage-class-name>.storageclass.storage.k8s.io/requests.storage`, so those will pretty much always have the raw name unless we do some regex matching or something. Plus it doesn't match what's in the YAML or CLI or docs, so it's harder to know what to change if you want to edit something. We had a number of bugs around trying to humanize quota keys in the previous console.

I think this is a losing battle, and we should just show the quota key as-is.

Reply:
Ok, thanks, sounds reasonable to me.  @cshinn, sound good to you?  -thanks

Reply:
yeah, that works for me. Especially if we're likely to have other things added